### PR TITLE
Andrew scoppa update caption

### LINKED ISSUE #5: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @AhmedTitef
- [ ] @AlexL890
- [ ] @JunRhee
- [ ] @KaKnife
- [ ] @Lionell72
- [ ] @advil0
- [ ] @andrew-scoppa
- [ ] @balsu79
- [ ] @cshiu0088
- [ ] @djgmtb
- [ ] @jennisang
- [ ] @kjlequire
- [ ] @llh41
- [ ] @skrishnan07
- [ ] @khanhhuytran
- [ ] @ochoa71
- [ ] @thenmoliGit

### PR DESCRIPTION
**Pull request by andrew-scoppa**
This PR closes #5

### CODE REVIEW

File: _slides/02-andrew-scoppa.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 layout: slide
 title: "Welcome to our slide deck!"
+author: Andrew
 ---
 
-Caption here
+_Cat Riddler_
```
Comment (kjlequire) [inline _slides/02-andrew-scoppa.md:7]: Change to Dog.

Comment (andrew-scoppa) [reply _slides/02-andrew-scoppa.md:7]: Why?

Comment (kjlequire) [reply _slides/02-andrew-scoppa.md:7]: I don't like cats.

**Review by kjlequire — CHANGES_REQUESTED** [review]
Pleas change to DOG

**Review by andrew-scoppa — COMMENTED** [review]
Changed cat to dog as per @kjlequire

**Review by kjlequire — APPROVED** [review]
Thanks. 👍

**Pull request merged**